debian/rules

```diff
@@ -34,6 +34,9 @@ override_dh_auto_test:
 
 override_dh_auto_install:
 	DESTDIR=$(CURDIR)/debian/tmp ninja -C build-deb install
+	# Debian kmod has a conf file called mlx4.conf that unnecessarily
+	# tries to autoload mlx4_en, call our conf file rdma-mlx4.conf instead
+	mv $(CURDIR)/debian/tmp/etc/modprobe.d/mlx4.conf $(CURDIR)/debian/tmp/etc/modprobe.d/rdma-mlx4.conf
```

Comment:
We already open bug [1] on this issue in kmod package,  and i'll ask them to fix it.

https://bugs.launchpad.net/ubuntu/+source/kmod/+bug/1693503